xrpl4j-model/src/main/java/org/xrpl/xrpl4j/model/flags/Flags.java

```diff
@@ -1294,4 +1295,353 @@ public PaymentChannelClaimFlags build() {
+       */
+      public NfTokenMintFlags.Builder tfOnlyXRP(boolean tfOnlyXRP) {
+        this.tfOnlyXRP = tfOnlyXRP;
+        return this;
+      }
+
+      /**
+       * Set {@code tfTrustLine} to the given value.
+       *
+       * @param tfTrustLine A boolean value.
+       *
+       * @return The same {@link NfTokenMintFlags.Builder}.
+       */
+      public NfTokenMintFlags.Builder tfTrustLine(boolean tfTrustLine) {
+        this.tfTrustLine = tfTrustLine;
+        return this;
+      }
+
+      /**
+       * Set {@code tfTransferable} to the given value.
+       *
+       * @param tfTransferable A boolean value.
+       *
+       * @return The same {@link NfTokenMintFlags.Builder}.
+       */
+      public NfTokenMintFlags.Builder tfTransferable(boolean tfTransferable) {
+        this.tfTransferable = tfTransferable;
+        return this;
+      }
+
+      /**
+       * Build a new {@link NfTokenMintFlags} from the current boolean values.
+       *
+       * @return A new {@link NfTokenMintFlags}.
+       */
+      public NfTokenMintFlags build() {
+        return NfTokenMintFlags.of(tfFullyCanonicalSig, tfBurnable, tfOnlyXRP, tfTrustLine, tfTransferable);
+      }
+    }
+  }
+
+  /**
+   * A set of static {@link Flags} which can be set on {@link org.xrpl.xrpl4j.model.transactions.NfTokenCreateOffer}s.
+   */
+  public static class NfTokenCreateOfferFlags extends Flags {
+
+    /**
+     * Constant {@link NfTokenCreateOfferFlags} for the {@code tfSellToken} flag.
+     */
+    public static final NfTokenCreateOfferFlags SELL_TOKEN = new NfTokenCreateOfferFlags(0x00000001);
+
+    private NfTokenCreateOfferFlags(long value) {
+      super(value);
+    }
+
+    /**
+     * Create a new {@link NfTokenCreateOfferFlags.Builder}.
+     *
+     * @return A new {@link NfTokenCreateOfferFlags.Builder}.
+     */
+    public static NfTokenCreateOfferFlags.Builder builder() {
+      return new NfTokenCreateOfferFlags.Builder();
+    }
+
+    private static NfTokenCreateOfferFlags of(boolean tfFullyCanonicalSig, boolean tfSellToken) {
+      return new NfTokenCreateOfferFlags(
+        TransactionFlags.of(
+          tfFullyCanonicalSig ? TransactionFlags.FULLY_CANONICAL_SIG : UNSET,
+          tfSellToken ? SELL_TOKEN : UNSET
+        ).getValue()
+      );
+    }
+
+    /**
+     * Construct {@link NfTokenCreateOfferFlags} with a given value.
+     *
+     * @param value The long-number encoded flags value of this {@link NfTokenCreateOfferFlags}.
+     *
+     * @return New {@link NfTokenCreateOfferFlags}.
+     */
+    public static NfTokenCreateOfferFlags of(long value) {
+      return new NfTokenCreateOfferFlags(value);
+    }
+
+    /**
+     * Require a fully canonical transaction signature.
+     *
+     * @return {@code true} if {@code tfFullyCanonicalSig} is set, otherwise {@code false}.
+     */
+    public boolean tfFullyCanonicalSig() {
+      return this.isSet(TransactionFlags.FULLY_CANONICAL_SIG);
+    }
```

Comment:
Once `NfTokenCreateOfferFlags` extends `TransactionFlags`, you can get rid of this method, since it is already defined in `TransactionFlags`